src/openforms/js/components/admin/form_design/variables/ServiceFetchConfigurationForm.stories.mdx

```diff
@@ -0,0 +1,72 @@
+import {ArgsTable, Canvas, Meta, Story} from '@storybook/addon-docs';
+
+import {FormLogicContext} from '../Context';
+import ServiceFetchConfigurationForm from './ServiceFetchConfigurationForm';
+
+<Meta title="Form design/ServiceFetchConfigurationForm" component={ServiceFetchConfigurationForm} />
+
+# ServiceFetchConfigurationForm
+
+Form to edit/create a ServiceFetchConfiguration
+
+export const Template = args => (
+  <FormLogicContext.Provider
+    value={{
+      services: [
+        {
+          label: 'Service 1',
+          apiRoot: 'http://foo.com/api/v1/',
+          apiType: 'ORC',
+        },
+        {
+          label: 'Service 2',
+          apiRoot: 'http://bar.com/api/v1/',
+          apiType: 'ORC',
+        },
+      ],
+    }}
+  >
+    <ServiceFetchConfigurationForm {...args} />;
+  </FormLogicContext.Provider>
+);
+
+## Presentation
+
+<Canvas>
+  <Story
+    name="Default"
+    args={{
+      formik: {
+        values: {
+          method: 'GET',
+          service: '',
+          path: '',
+          queryParams: [['', ['']]],
+          headers: [['', '']],
+          body: '',
+          dataMappingType: '',
+          mappingExpression: '',
+        },
+        onSubmit: (values, {setSubmitting}) => {
+          alert(JSON.stringify(values, null, 2));
+        },
+        getFieldProps: ({...args}) => null,
+        getFieldHelpers: name => ({}),
+      },
+      stateData: {
+        service: 'http://openforms.dev/services/foo',
+        path: '/foo',
+        method: 'POST',
+      },
+      setData: ({...args}) => {
+        null;
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+### Props
+
+<ArgsTable story="Default" />
```

Comment:
```suggestion
<ArgsTable of={ServiceFetchCOnfigurationForm} />
```

Reply:
When using this for the `MappingArrayInput` it doesn't seem to work :thinking:
![image](https://user-images.githubusercontent.com/29249171/223487067-d6c87679-49a3-45d2-baf4-5474944693b0.png)